contracts/STokensManagerStorage.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MPL-2.0
+pragma solidity 0.8.4;
+
+import {ISTokensManager} from "@devprotocol/i-s-tokens/contracts/interface/ISTokensManager.sol";
+
+contract STokensManagerStorage {
```

Comment:
Storage is stored in the Proxy contract, so why need this contract?

Reply:
#1 (comment)

For the same reason.

Reply:
I will delete it
@aggre